styleguide/source/_patterns/02-molecules/page-filters.twig

```diff
@@ -0,0 +1,28 @@
+<div class="joe__filters-wrapper">
```

Comment:
The chosen.js library doesn't seem to be working quite right. When you reset the selections still appear along with part of the placeholder text.

![image](https://user-images.githubusercontent.com/394763/35400241-d254bd2e-01bb-11e8-947c-84af1e5bbeb4.png)

Reply:
I can't replicate this error.